core/src/filed/filed_utils.cc

```diff
@@ -74,7 +69,7 @@ bool CheckResources()
 #endif
     }
 
-    /* pki_encrypt implies pki_sign */
+    /* pki_encrypt fd_implies pki_sign */
```

Comment:
Im not sure what fd_implies stands for. Can you help me out ?

Reply:
`implies` is the right word.

Recently, there used to be an attribute of `jcr` called `impl`. This attribute was renamed to accommodate certain issues we had. It was renamed in a way to reflect which daemon it belonged to, and so we got `dir_impl`, `sd_impl` and `fd_impl`.
So my best guess here is that during some automatic refactoring, the `impl` part of `implies` was renamed too